### ISSUE TITLE
LostCities 1.12-2.0.8 Crash with Glacidus mod

### ISSUE THREAD

**Issue by CaptainStalfoss** [McJty/LostCities#131]
I was updating my mods and noticed i can't go to the Glacidus dimension with LostCities 1.12-2.0.8 active,
it crashes the game.
I reverted back to LostCities 1.12-2.0.7 and it works fine.
something in 2.0.8 is causing it.

(the game only crashes if you have a lostcities profile loaded, if its set to default minecraft, it wont crash.)

here's the crash:
[crash.txt](https://github.com/McJtyMods/LostCities/files/2062670/crash.txt)
yes i had CustomNPC mod on, but i doubt that mod is the cause here.

**Comment by McJty** [comment]
Did you modify configs? What Dimension ID is Glacidus using?

**Comment by CaptainStalfoss** [comment]
Glacidus Dim84
Lostcities Dim99

**Comment by McJty** [comment]
Dim99? So you did modify the lost city config? Maybe that's what is causing the problem. Can you show it to me? (the general.cfg)

**Comment by CaptainStalfoss** [comment]
yes i did, due to a mod conflict. (but i disabled those mods for the crash test)
[config.txt](https://github.com/McJtyMods/LostCities/files/2062709/config.txt)
